hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/SerialConsumer.java

```diff
@@ -96,13 +100,19 @@ private int calculateInflightSize(Subscription subscription) {
     @Override
     public void consume(Runnable signalsInterrupt) {
         try {
+            ConsumerProfiler profiler = subscription.isProfilingEnabled() ? new DefaultConsumerProfiler(subscription.getQualifiedName()) : new NoOpConsumerProfiler();
+            profiler.startMeasurements(Measurement.SIGNALS_AND_SEMAPHORE_ACQUIRE);
             do {
                 loadRecorder.recordSingleOperation();
+                profiler.startPartialMeasurement(Measurement.SIGNALS_INTERRUPT_RUN);
```

Comment:
Actually all of the `signalsInterrupt.run()` will be instrumented, i have checked in and it will look like this.
```
partialMeasurements: 
signalsInterrupt.run = StopWatch 'signalsInterrupt.run': 43.347667 milliseconds
--------------------------------------------------------
Milliseconds  %       Task name
--------------------------------------------------------
30.832625     71%     signalsInterrupt.run
12.515042     29%     signalsInterrupt.run
```